src/main/resources/digital/slovensko/autogram/ui/gui/settings-dialog.fxml

```diff
@@ -236,6 +236,41 @@
                                             styleClass="autogram-smaller-radio-buttons" />
                                     </VBox>
                                 </HBox>
+                                <HBox styleClass="autogram-settings-row">
+                                    <VBox styleClass="left">
+                                        <TextFlow>
+                                            <Text styleClass="autogram-heading-s">
+                                                Absolútna cesta k podpisovému keystore
+                                            </Text>
+                                        </TextFlow>
+                                        <TextFlow>
+                                            <Text styleClass="autogram-description">
+                                                Absolútna cesta ku keystore súboru na disku, ktorý sa má použiť na podpisovanie.
+                                            </Text>
+                                        </TextFlow>
+                                    </VBox>
+                                    <VBox styleClass="autogram-textfield-container">
+                                        <TextField fx:id="customKeystorePathTextField" styleClass="autogram-input" />
+                                    </VBox>
+                                </HBox>
+                                <HBox styleClass="autogram-settings-row">
+                                    <VBox styleClass="left">
+                                        <TextFlow>
+                                            <Text styleClass="autogram-heading-s">
+                                                Vyžadovať heslo k podpisovému keystore
+                                            </Text>
+                                        </TextFlow>
+                                        <TextFlow>
+                                            <Text styleClass="autogram-description">
+                                                Ak má keystore nastavené neprázdne heslo, zapnite toto nastavenie.
```

Comment:
Toto sa neda nejako zistit zvonka?

Reply:
Java chce pri všetkých heslo - hoci aj prázdne. Nedá sa to zistiť.